toolchain/check/handle_function.cpp

```diff
@@ -211,8 +211,12 @@ static auto BuildFunctionDecl(Context& context,
     return_slot = SemIR::Function::ReturnSlot::Absent;
   }
 
-  SemIR::InstBlockId param_refs_id =
-      context.node_stack().Pop<Parse::NodeKind::TuplePattern>();
+  auto param_refs_id =
+      context.node_stack().PopIf<Parse::NodeKind::TuplePattern>();
+  if (!param_refs_id) {
+    context.TODO(node_id, "function with positional parameters");
```

Comment:
I'm confused by this, isn't this an absence of positional params?

Reply:
Per #3848, ["positional parameters"](https://github.com/carbon-language/carbon-lang/blob/ca4be05c9aa31ae0866ff491a32e1f12530cf6d5/proposals/p3848.md#positional-parameters) are introduced by the absence of an explicit parameter list. Eg:
```
fn F => $0 < $1;
```
... has so-called positional parameters `$0` and `$1`, which are introduced because `F` doesn't have an explicit parameter list.